### PR TITLE
Us 709 voltage level dialog

### PR DESCRIPTION
**Pull request by laurgarn**

### CODE REVIEW

File: src/components/dialogs/load-creation-dialog.js [hunk 1]
```diff
@@ -348,19 +348,21 @@ const LoadCreationDialog = ({
                                 errors.get('voltage-level')?.error
                             }
                             helperTextVoltageLevel={
-                                errors.get('voltage-level')?.error &&
-                                intl.formatMessage({
-                                    id: errors.get('voltage-level')?.errorMsgId,
-                                })
+                                makeErrorHelper(errors, intl, 'voltage-level')
+                                // errors.get('voltage-level')?.error &&
```
Comment (JacquesBorsenberger) [inline src/components/dialogs/load-creation-dialog.js:352]: remove commented code

File: src/components/dialogs/load-creation-dialog.js [hunk 2]
```diff
@@ -348,19 +348,21 @@ const LoadCreationDialog = ({
...
                             errorBusOrBusBarSection={
                                 errors.get('bus-bar')?.error
                             }
                             helperTextBusOrBusBarSection={
-                                errors.get('bus-bar')?.error &&
-                                intl.formatMessage({
-                                    id: errors.get('bus-bar')?.errorMsgId,
-                                })
+                                makeErrorHelper(errors, intl, 'bus-bar-level')
+                                // errors.get('bus-bar')?.error &&
```
Comment (JacquesBorsenberger) [inline src/components/dialogs/load-creation-dialog.js:362]: remove commented code

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/components/dialogs/voltage-level-creation-dialog.js [hunk 3]
```diff
@@ -0,0 +1,761 @@
...
+    function makeBusBarSelector(cnx, isFrom) {
+        let fieldName = isFrom ? 'fromBBS' : 'toBBS';
+        let prefix = 'cnx_' + fieldName + '.' + cnx.idx;
+        let value = cnx[fieldName];
+        let sideBlock = makeBlock(prefix, value);
+        let bbsCb = (event, value) => {
+            onConnectivityChange(cnx.idx, value, fieldName);
+        };
+        let bbs = isFrom ? cnx.fromBBS : cnx.toBBS;
+        let af = isFrom && !cnx.fromBBS;
```
Comment (JacquesBorsenberger) [inline src/components/dialogs/voltage-level-creation-dialog.js:648]: again af is not enough explicit

**Review by JacquesBorsenberger — CHANGES_REQUESTED** [review]
remove commented code, and maybe extract connexion / busbar to their own components

File: src/components/dialogs/voltage-level-creation-dialog.js [hunk 4]
```diff
@@ -0,0 +1,761 @@
...
+                </Button>
+            </DialogActions>
+        </Dialog>
+    );
+
+    function makeTFld(
+        fieldVarInBlock,
+        onChangeCb,
+        labelId,
+        af = false,
```
[the remark this answers is not in the document]
Comment (laurgarn) [reply src/components/dialogs/voltage-level-creation-dialog.js]: 'af' expanded to 'autoFocus'

File: src/components/dialogs/load-creation-dialog.js (hunk 2 above)
Comment (laurgarn) [reply src/components/dialogs/load-creation-dialog.js]: done

File: src/components/dialogs/load-creation-dialog.js (hunk 1 above)
Comment (laurgarn) [reply src/components/dialogs/load-creation-dialog.js]: done

[the remark this answers is not in the document]
Comment (laurgarn) [reply src/components/dialogs/load-creation-dialog.js]: done

File: src/components/dialogs/load-creation-dialog.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (laurgarn) [reply src/components/dialogs/load-creation-dialog.js]: done

File: src/translations/network-modifications-fr.json [hunk 5]
```diff
@@ -1,6 +1,7 @@
 {
     "network_modification/modificationsCount": "{count, plural, =0 {aucune modification} =1 {# modification} other {# modifications}}",
     "network_modifications/EQUIPMENT_DELETION": "Suppression de {computedLabel}",
+    "network_modifications/SUBSTATION_CREATION": "Création du site {computedLabel}",
```
[the remark this answers is not in the document]
Comment (laurgarn) [reply src/translations/network-modifications-fr.json:4]: done

File: src/translations/network-modifications-en.json [hunk 6]
```diff
@@ -1,6 +1,7 @@
 {
     "network_modification/modificationsCount": "{count, plural, =0 {no modification} =1 {{count} modification} other {{count} modifications}}",
     "network_modifications/EQUIPMENT_DELETION": "Deletion of {computedLabel}",
+    "network_modifications/SUBSTATION_CREATION": "Creating substation {computedLabel}",
```
[the remark this answers is not in the document]
Comment (laurgarn) [reply src/translations/network-modifications-en.json:4]: done

File: src/components/dialogs/voltage-level-creation-dialog.js (hunk 3 above)
Comment (laurgarn) [reply src/components/dialogs/voltage-level-creation-dialog.js]: 'af' expanded to 'autoFocus'

**Review by JacquesBorsenberger — APPROVED** [review]
still think that busbar and connexion shoud have their own widget but it is ok

**Review by FranckLecuyer — APPROVED** [review]

**Pull request merged**